docs/whats-new/dotnet-2020-01.md

```diff
@@ -0,0 +1,172 @@
+ms.date: 02/02/2020
+---
+
+# .NET docs: What's new for January 2020
+
+Welcome to what's new in .NET docs for January 2020. This article lists some of the major changes to docs during this period.
+
+## Architecture guides
+
+### New articles
+
+- [Caching in a cloud-native app](../architecture/cloud-native/azure-caching.md) - Chapter 5 octupdates
+- [Database-per-microservice](../architecture/cloud-native/database-per-microservice.md) - Chapter 5 octupdates
+- [Elasticsearch in a cloud-native app](../architecture/cloud-native/elastic-search-in-azure.md) - Chapter 5 octupdates
+
+## .NET Core
+
+### New articles
+
+- [APIs that always throw exceptions on .NET Core](../core/compatibility/unsupported-apis.md) - Unsupported APIs
+- [Fedora 31 Package Manager - Install .NET Core](../core/install/linux-package-manager-fedora31.md) - Add fedora 31;rearrange package manager links
+- [Ubuntu 19.10 Package Manager - Install .NET Core](../core/install/linux-package-manager-ubuntu-1910.md) - Add ubuntu 19.10
+- [How to port a C++/CLI project to .NET Core](../core/porting/cpp-cli.md) - Add NetFx -> NetCore porting documentation for C++/CLI projects
+
+### Updated articles
+
+- [Breaking changes for migration from Version 2.2 to 3.0](../core/compatibility/2.2-3.0.md) - Add bookmark indexes to breaking changes docs
+- [Breaking changes for migration from Version 2.2 to 3.1](../core/compatibility/2.2-3.1.md) - Add bookmark indexes to breaking changes docs
+- [Breaking changes for migration from .NET Framework to .NET Core](../core/compatibility/fx-core.md) - Breaking changes improvements
+- [Changes that affect compatibility](../core/compatibility/index.md) - Add emoji text where needed
+- [Run-time configuration options for compilation](../core/run-time-config/compilation.md) - Add MSBuild props for run-time config
+- [Run-time configuration options for garbage collection](../core/run-time-config/garbage-collector.md) - GC config knobs touch-ups
+- [global.json overview](../core/tools/global-json.md) - Update global.json for 3.0
+
+## C# language
+
+### New articles
+
+- [Value types (C# reference)](../csharp/language-reference/builtin-types/value-types.md) - Moved and revised the value types article
+- [\<inheritdoc> (C# Programming Guide)](../csharp/programming-guide/xmldoc/inheritdoc.md) - Added new inheritdoc xml documentation
+
+### Updated articles
+
+- [Boxing and Unboxing (C# Programming Guide)](../csharp/programming-guide/types/boxing-and-unboxing.md) - Clarify CLR acronym
+- [\<summary> (C# programming guide)](../csharp/programming-guide/xmldoc/summary.md) - Update XML documentation comments
+
+## .NET Framework
+
+### Updated articles
+
+- [Runtime Directives (rd.xml) Configuration File Reference](../framework/net-native/runtime-directives-rd-xml-configuration-file-reference.md) - Fix formatting of rd.xml documentation
+- [Unsupported scenarios](../framework/wcf/feature-details/unsupported-scenarios.md) - Acrolinx 12/30
```

Comment:
I thought we were ignoring Acrolinx PRs? This one mostly changed trailing white spaces.

```suggestion
```

Reply:
Hmm. I thought we had too. I must not have merged that change yet. Good catch.